### PR TITLE
AP_AHRS: rotate type=10 wind to correct direction 

### PR DESCRIPTION
**Pull request by peterbarker**
also contains a fix for SIM-on-Hardware for the barometer always being nullptr in the sim.

### CODE REVIEW

File: libraries/AP_AHRS/AP_AHRS.cpp [hunk 1]
```diff
@@ -779,6 +779,7 @@ Vector3f AP_AHRS::wind_estimate(void) const
         if (_sitl) {
             const auto &fdm = _sitl->state;
             wind = fdm.wind_ef;
+            wind.rotate_xy(radians(180));
```
Comment (tridge) [inline libraries/AP_AHRS/AP_AHRS.cpp:782]: this should be fixed in the source where it is set in fdm, which backend gets this wrong?
changing it here would almost certainly break RealFlight, XPlane and others

**Review by tridge — CHANGES_REQUESTED** [review]

File: libraries/AP_AHRS/AP_AHRS.cpp (hunk 1 above)
Comment (peterbarker) [reply libraries/AP_AHRS/AP_AHRS.cpp:782]: > this should be fixed in the source where it is set in fdm, which backend gets this wrong? changing it here would almost certainly break RealFlight, XPlane and others

The default simulation.  You can simulate it yourself - start a Plane sim normally, flip to backend type 10 and you will see the wind spin around 180-degrees.  I'll work up a patch to change it just in the affected backend.

Comment (tridge) [reply libraries/AP_AHRS/AP_AHRS.cpp:782]: i think it is also worth bisecting this, I'm sure they were consistent in the past

**Comment by peterbarker** [comment]
This wasn't the right way to do things.

Closing in favor of other PR.

**Pull request closed without merging**